app/controllers/users_controller.rb

```diff
@@ -0,0 +1,61 @@
+class UsersController < ApplicationController
+  before_action :set_user, only: [:edit, :update, :show]
+  before_action :require_same_user, only: [:edit, :update]
+
+  def new
+    cant_register_after_login
+
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+
+    if @user.save
+      session[:user_id] = @user.id
+      flash[:success] = "Your profile has been created!"
+      redirect_to login_path
+    else
+      render :new
+    end
+  end
+
+  def edit
```

Comment:
There are usually 2 ways how to do this - `def edit; end` - as 1 liner or just delete it. (I prefer the former way)